plugins/Ecommerce/templates/getSparklines.twig

```diff
@@ -1,70 +1,157 @@
+            <img style="padding-right:4px;padding-left:4px;margin-top:-2px" src="plugins/MultiSites/images/{{ trend_icon }}.png">
+            <strong class="{{ trend_class }}-evolution">{{ trend_prefix }}{{ revenue_trend_percent|raw }}</strong>
+        </div>
+        {% endif %}
     </div>
 
-    <div class="sparkline">{{ sparkline(urlSparklineAverageOrderValue) }}
-	<div>
-        <strong>{{ avg_order_revenue|money(idSite)|raw }}</strong>
-        {{ 'General_AverageOrderValue'|translate }}
-	</div>
+    <div class="sparkline">
+        {{ sparkline(urlSparklineAverageOrderValue) }}
+        <div class="sparkline-metrics">
+            <strong>{{ avg_order_revenue|money(idSite)|raw }}</strong>
+            {{ 'General_AverageOrderValue'|translate }}
+        </div>
+        {% if avg_order_revenue_trend is defined %}
+        <div class="metricEvolution" title="{{ avg_order_revenue_tooltip|raw }}">
+            {% if avg_order_revenue_trend == -1 %} {% set trend_icon='arrow_down' %} {% set trend_class='negative' %} {% set trend_prefix='' %}  {% endif %}
+            {% if avg_order_revenue_trend == 1 %} {% set trend_icon='arrow_up' %} {% set trend_class='positive' %} {% set trend_prefix='+' %} {% endif %}
+            {% if avg_order_revenue_trend == 0 %} {% set trend_icon='stop' %} {% set trend_class='neutral' %} {% set trend_prefix='' %} {% endif %}
+            <img style="padding-right:4px;padding-left:4px;margin-top:-2px" src="plugins/MultiSites/images/{{ trend_icon }}.png">
+            <strong class="{{ trend_class }}-evolution">{{ trend_prefix }}{{ avg_order_revenue_trend_percent|raw }}</strong>
+        </div>
+        {% endif %}
     </div>
+
 </div>
+
 <div id='leftcolumn' {% if not isWidget %}style='width:33%;'{% endif %}>
-    <div class="sparkline">{{ sparkline(urlSparklineConversionRate) }}
-	<div>
-        {% set ecommerceOrdersText %}{{ 'General_EcommerceOrders'|translate }}{% endset %}
-        {{ 'Goals_ConversionRate'|translate("<strong>"~conversion_rate|percent~"</strong> "~ecommerceOrdersText)|raw }}
-	</div>
+
+    <div class="sparkline">
+        {{ sparkline(urlSparklineConversionRate) }}
+        <div class="sparkline-metrics">
+            {% set ecommerceOrdersText %}{{ 'General_EcommerceOrders'|translate }}{% endset %}
+            {{ 'Goals_ConversionRate'|translate("<strong>"~conversion_rate|percent~"</strong> "~ecommerceOrdersText)|raw }}
+        </div>
+        {% if conversion_rate_trend is defined %}
+        <div class="metricEvolution" title="{{ conversion_rate_tooltip|raw }}">
+            {% if conversion_rate_trend == -1 %} {% set trend_icon='arrow_down' %} {% set trend_class='negative' %} {% set trend_prefix='' %}  {% endif %}
+            {% if conversion_rate_trend == 1 %} {% set trend_icon='arrow_up' %} {% set trend_class='positive' %} {% set trend_prefix='+' %} {% endif %}
```

Comment:
To be fully correct you need to use `'Intl_NumberSymbolPlus'|translate` instead of a simple `+`, as some languages use a custom plus sign

Reply:
Good point, thanks!